src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -125,8 +125,34 @@ protected override void SetupNewTextView(IVsTextView textView)
             var openDocument = wpfTextView.TextBuffer.AsTextContainer().GetRelatedDocuments().FirstOrDefault();
             var isOpenMetadataAsSource = openDocument != null && openDocument.Project.Solution.Workspace.Kind == WorkspaceKind.MetadataAsSource;
 
+            ConditionallyCollapseOutliningRegions(textView, wpfTextView, optionsService, isOpenMetadataAsSource);
```

Comment:
Diff looks much larger than it is.  All i did was add a null check.  But i then extracted out the check and the previous collapsing code into a new method ```ConditionallyCollapseOutliningRegions```.